slk/repl/repl.py

```diff
@@ -1032,11 +1032,47 @@ def do_new_account(self: SidechainRepl, line: str) -> None:
             return
         args.pop(0)
 
-        for alias in args:
+        # an new account either generates its own new secret seed, or one may be
+        # specified by using a `-s <seed>` switch
+        aliases = []
+        seeds = []
+        index = 0
+        while True:
+            if index == len(args):
+                break
+            if args[index] == '-s':
+                print(
+                    "Error: new_account -s switch must come after the alias argument. Type "
+                    '"help" for help.'
+                )
+            aliases.append(args[index])
+            seeds.append(None)
+            index += 1
+            if index == len(args):
+                break
+            if args[index] != '-s':
+                continue
+            # user supplied seed
+            index += 1
+            if index == len(args):
+                print(
+                    "Error: new_account -s switch takes one argument. Type "
+                    '"help" for help.'
+                )
```

Comment:
ditto

Reply:
Fixed in `491c3e8 fixup! feat: Add load_batch command (#89)`